opengrok-tools/src/main/python/opengrok_tools/utils/webutil.py

```diff
@@ -27,45 +27,23 @@
 
 
 def get(logger, uri, params=None, headers=None):
-    try:
-        proxies = get_proxies(uri)
-        return requests.get(uri, params=params, proxies=proxies)
-    except Exception:
-        logger.debug(traceback.format_exc())
-        return None
+    proxies = get_proxies(uri)
```

Comment:
A lot of repetitions here, wouldn't it make sense to factor the common code into a single function (named, say `request`) that all verbs would call?

Reply:
refactored. I could possibly go as far as making `do_api_call()` to use `requests.__dict__` to lookup the function names.